From: Anjali Singhai Jain <anjali.singhai@intel.com>
Date: Thu, 22 May 2014 06:32:17 +0000
Subject: [PATCH 103/144] i40e/i40evf: Add Flow director stats to PF stats
Patch-mainline: v3.16-rc1
Git-commit: 433c47de1368cf07a7cb8d9148a4d36d824b89f8 (partial)
References: bsc#909486 FATE#317393

Add members to stat struct to keep track of Flow director ATR and
SideBand filter packet matches.

Change-ID: Ibbb31a53c7adcc2bb96991dd80565442a2f2513c
Signed-off-by: Anjali Singhai Jain <anjali.singhai@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: David Chang <dchang@suse.com>
---
 drivers/net/ethernet/intel/i40evf/i40e_type.h |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/ethernet/intel/i40evf/i40e_type.h
+++ b/drivers/net/ethernet/intel/i40evf/i40e_type.h
@@ -1023,6 +1023,9 @@ struct i40e_hw_port_stats {
 	u64 tx_size_big;		/* ptc9522 */
 	u64 mac_short_packet_dropped;	/* mspdc */
 	u64 checksum_error;		/* xec */
+	/* flow director stats */
+	u64 fd_atr_match;
+	u64 fd_sb_match;
 	/* EEE LPI */
 	u32 tx_lpi_status;
 	u32 rx_lpi_status;
